### ISSUE TITLE
Lesson 1 - Solidity Basics

### ISSUE THREAD

**Issue by BoxedFruits** [BoxedFruits/KaijuKata#3]
Should go over basic syntax as well as some datatypes of Solidity

datatypes:
uint
int
bytes
address
string
bool
(possibly arrays?)

access:
public
private
constant
immutable

**Comment by SamarthhD** [comment]
Yes Arrays is an equally important data type . After understanding Arrays, one can learn conditionals and  loops  in a better way  and then proceed on to other reference data types like strings and mappings in Solidity.
